ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -164,10 +155,35 @@ def get_winservices_modes(self):
             if svc.monitored():
                 data[svc.id] = {'modes': svc.get_serviceclass_startmodes(),
                                 'mode': svc.startMode,
-                                'monitor': svc.monitored()
+                                'monitor': svc.monitored(),
+                                'severity': svc.getFailSeverity(),
                                 }
         return data
 
+    def getMonitoredDataSource(self):
+        '''Return datasource for template if it exists'''
+        template = self.getRRDTemplate()
+        if template:
+            for datasource in template.getRRDDataSources():
```

Comment:
Because of the order of precedence that we've outlined, you'll need to check specifically for the 'DefaultService' first, then loop through the rest if necessary.